docs/src/theme/Layout/styles.module.css

```diff
@@ -0,0 +1,22 @@
+html,
+body {
+  height: 100%;
+}
+
+.mainWrapper {
+  display: grid;
```

Comment:
Not sure if it's caused by changing to a grid system, but some pages are now broken in the mobile perspective.

See the "Middleware" page as an example:
https://robo-git-fork-mbos2-docs-ui-update-wave-play.vercel.app/docs/advanced/middleware

<img width="393" alt="Screenshot 2023-12-23 at 10 09 24 PM" src="https://github.com/Wave-Play/robo.js/assets/3953360/c93a0f6e-ddcd-4690-bb44-477736e6f8a8">

Looks like the page width is wrongly computed this way. :/

Reply:
That should not be the cause because that wrapper is not present in docs pages, only in Home page (Home part)

Reply:
Actually that is the cause of it while testing. 

Will find another solution for this.

Reply:
Awesome, that looks like it worked!